blockchain/treasury_test.go

```diff
@@ -1844,6 +1844,262 @@ func TestTSpendExpendituresPolicy(t *testing.T) {
+	// for the fix sequence locks agenda, and, finally, ensure it is always
+	// available to vote by removing the time constraints to prevent test
+	// failures when the real expiration time passes.
+	const tVoteID = chaincfg.VoteIDTreasury
+	params = cloneParams(params)
+	tVersion, deployment, err := findDeployment(params, tVoteID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	removeDeploymentTimeConstraints(deployment)
+
+	// Dave off tvi and mul.
+	tvi := params.TreasuryVoteInterval
+	mul := params.TreasuryVoteIntervalMultiplier
+
+	// Create a test harness initialized with the genesis block as the tip.
+	g, teardownFunc := newChaingenHarness(t, params, "treasurytest")
+	defer teardownFunc()
+
+	// Helper to check the treasury balance at tip.
+	assertTipTreasuryBalance := func(wantBalance int64) {
+		t.Helper()
+		ts, err := getTreasuryState(g, g.Tip().BlockHash())
+		if err != nil {
+			t.Fatal(err)
+		}
+		if ts.Balance != wantBalance {
+			t.Fatalf("unexpected treasury balance. want=%d got %d",
+				wantBalance, ts.Balance)
+		}
+	}
+
+	// replaceTreasuryVersions is a munge function which modifies the
+	// provided block by replacing the block, stake, and vote versions with
+	// the treasury deployment version.
+	replaceTreasuryVersions := func(b *wire.MsgBlock) {
+		chaingen.ReplaceBlockVersion(int32(tVersion))(b)
+		chaingen.ReplaceStakeVersion(tVersion)(b)
+		chaingen.ReplaceVoteVersions(tVersion)(b)
+	}
+
+	// ---------------------------------------------------------------------
+	// Generate and accept enough blocks with the appropriate vote bits set
+	// to reach one block prior to the treasury agenda becoming active.
+	// ---------------------------------------------------------------------
+
+	g.AdvanceToStakeValidationHeight()
+	g.AdvanceFromSVHToActiveAgenda(tVoteID)
+
+	// Ensure treasury agenda is active.
+	tipHash := &g.chain.BestSnapshot().Hash
+	gotActive, err := g.chain.IsTreasuryAgendaActive(tipHash)
+	if err != nil {
+		t.Fatalf("IsTreasuryAgendaActive: %v", err)
+	}
+	if !gotActive {
+		t.Fatalf("IsTreasuryAgendaActive: expected enabled treasury")
+	}
+
+	// tbaseBlocks will keep track of how many blocks with a treasury base
+	// have been added to the chain.
+	var tbaseBlocks int
+
+	// ---------------------------------------------------------------------
+	// Generate enough blocks to have spendable outputs during the reorg
+	// stage of the test. Due to having lowered the coinbase maturity,
+	// we'll need to manually generate and store these outputs until we
+	// have enough to create two sidechains in the future.
+	//
+	//   ... -> bva19 -> bout0 -> .. boutn
+	// ---------------------------------------------------------------------
+	outs := g.OldestCoinbaseOuts()
+	neededBlocks := (params.CoinbaseMaturity + 1) * 2
+	neededOuts := neededBlocks * params.TicketsPerBlock
+	oldOuts := make([][]chaingen.SpendableOut, 1)
+	for i := uint32(0); i < uint32(neededOuts); i++ {
+		name := fmt.Sprintf("out%v", i)
```

Comment:
name doesn't match the comment of the scheme used throughout.  Should be `bout%v`